custom_components/powercalc/data/ewelight/ZB-CL01/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly 2.5 gen1 (SHSW-25)",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.10.6:docker"
+    },
+    "name": "ZB-CL01",
```

This may be the "marketed" name.
```suggestion
    "name": "RGBCW 7W E27/B22 Bulb",
```

---

The name discoverd by zigbee is "eWeLight ZB-CL01"

The package says "RGBCW 5W GU10 smart Bulb"

---

```
min_color_temp_kelvin: 2702
max_color_temp_kelvin: 6535
min_mireds: 153
max_mireds: 370
effect_list: colorloop
supported_color_modes: color_temp, xy
effect: null
color_mode: color_temp
brightness: 56
color_temp_kelvin: 3952
color_temp: 253
hs_color: 26.835, 35.746
rgb_color: 255, 204, 163
xy_color: 0.424, 0.366
off_with_transition: false
off_brightness: null
friendly_name: eWeLight ZB-CL01
supported_features: 44
```

---

https://zigbee.blakadder.com/eWeLight_ZYZB007.html

---

There is a FCC ID on the package : [2APYC-JL01](https://fccid.io/2APYC-JL01)
It's registered as JULUN Technology Lighting CO.,LTD Smart light JL01